### PR TITLE
HTML fundamentals Assignment

### PR DESCRIPTION
**Pull request by ShaikMoAbdullah**
- [x] Completed Poem Webpage task
Screenshot 2023-08-05 at 11 24 21 PM

- [x] Completed Hometown Webpage task
Screenshot 2023-08-05 at 11 24 05 PM

- [x] Completed Your Favorite Fan Webpage task
Screenshot 2023-08-05 at 11 23 38 PM

- [x] Completed HTML Tables task
Screenshot 2023-08-05 at 11 23 18 PM

- [x] Completed News Article Webpage task
Screenshot 2023-08-05 at 11 22 57 PM

### CODE REVIEW

File: Week-1/assignment_2.1/index.html
Comment (NishantNaithani) [inline Week-1/assignment_2.1/index.html:1]: Here always wrap your webpage content under a parent tag,
For e.g. `<main><p>......</p></main>`

And the expected result is different not matching with your result.

File: Week-1/assignment_2.2/index.html
Comment (NishantNaithani) [inline Week-1/assignment_2.2/index.html:1]: Here always wrap your webpage content under a parent tag,
For e.g. `<main><p>......</p></main>`

**Comment by NishantNaithani** [comment]
@ShaikMoAbdullah ,

Added comments.

Thanks

**Review by NishantNaithani — CHANGES_REQUESTED** [review]

File: Week-1/assignment_2.1/index.html
Comment (ShaikMoAbdullah) [reply Week-1/assignment_2.1/index.html:1]: Thanks for informing. I didn't know it should look the same. I missed this line, `Maintain the correct formatting` given in the LMS. 

Will make the required changes.

File: Week-1/assignment_2.2/index.html
Comment (ShaikMoAbdullah) [reply Week-1/assignment_2.2/index.html:1]: Will make the required changes

**Comment by ShaikMoAbdullah** [comment]
Made the required changes

Thanks

**Review by NishantNaithani — APPROVED** [review]

**Pull request merged**